source/playbooks/refining-and-improving.rst

```diff
@@ -32,6 +32,17 @@ Use the run timeline to help write an accurate retrospective. Events such as own
 .. image:: ../images/Retro.gif
    :alt: Create and publish retrospective reports.
 
+Metrics
+~~~~~~~
+
+Use metrics to identify key areas where you want to extract valuable insights by measuring performance and improvement. Metrics are enabled when you enable retrospectives. Calibrate the type of metric you want to measure, and once a run is finished you can view the output in the retrospective. You can have multiple metrics configured per playbook and you can edit them at any time. Metrics can be configured based on numeric, time, or value input.
+
+These metrics can be anything that's of interest to you and your team. For example, for a software release playbook you might want to have a metric tracking how many bugs were detected during a run. The output of the metrics you've added is provided in the retrospective. Over time, you can use metrics across retrospectives to examine anomalies and refine goals.
+
+Another example is a support incident playbook. The time to resolution metric can be applied and used to identify areas that need more refinement, such as tasks that might work better if they're split up so goals are reached faster.
+
+When you delete configured metrics from a playbook, you will no longer see those metrics in the dashboard. In addition, the corresponding metric field will be removed from the retrospective form and from published retrospectives.
```

Comment:
@gbochora, one more question - I'd imagine that if someone inadvertently deleted a metric and it disappeared from all retros, they may have a problem, especially if it's a valuable one.

Should we encourage/suggest that retros are downloaded/copied so data doesn't get lost? I don't want to set users up for failure/data loss. :)

Reply:
Data will not be lost, it will still be stored in the database. Currently, there is no way to export it, but we are planning to add such functionality as far as I know.
Also, one question, should we add more information about the playbook metrics dashboard?

Reply:
Great suggestion; do you have anything specific in mind or any specs I can look at?

Reply:
This is from Epic description(just updated few details):
For each run, participants can enter an actual value per metric under the retrospective section. They can edit it as many times as they'd like until publishing the retrospective at which point both the metrics and report will no longer be editable. The final value only shows up in the playbook dashboard once retrospective has been published.

The playbook dashboard reports on each metric for all runs by showing:

- average value for all runs
- last 10-run average value and difference with prev 10-run average
- value range for all runs
- target value
- actual values chart for last 10-runs

The lower section shows a list of finished runs with metrics values. Customers can filter or sort a list by metrics values.

Please let me know if you have any questions.

Design:
![image](https://user-images.githubusercontent.com/4368372/157894280-1a757f96-fd8e-42b9-9f0d-314dd97473a9.png)